client/state/index.js

```diff
@@ -109,7 +109,8 @@ const middleware = [ thunkMiddleware, noticesMiddleware ];
 if ( typeof window === 'object' ) {
 	// Browser-specific middlewares
 	middleware.push(
-		require( './analytics/middleware.js' ).analyticsMiddleware
+		require( './analytics/middleware.js' ).analyticsMiddleware,
+		require( './data-layer/wpcom-api-middleware.js' ).middleware,
```

Comment:
fair enough. we might still have a valid reason to use a faster async method on ssr. should not be a problem though. if we need to take this out of browser-only mode it's trivial